modules/ci_environment/files/check_rsyslog_status_transition.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if ! /usr/lib/nagios/plugins/check_file_age -f /srv/logs/log-1/cdn/bouncer_and_redirector.log -c600 -w300 > /dev/null
+then
+    logger "ERROR: /srv/logs/log-1/cdn/bouncer_and_redirector.log is older than 5mins. Restarting rsyslog..."
+    sudo service rsyslog restart
```

Comment:
if running as root we dont need sudo. 
Also we should use fall paths or set a PATH to make sure.